evap/evaluation/templatetags/morefilters.py

```diff
@@ -70,3 +70,19 @@ def render(self, context):
 @register.filter
 def is_false(arg): 
     return arg is False
+
+@register.filter
+def is_choice_field(field):
+    return field.field.__class__.__name__ == "TypedChoiceField"
+
+@register.filter
+def is_multiple_choice_field(field):
+    return field.field.__class__.__name__ == "MultipleChoiceField"
+
+@register.filter
+def is_textfield(field):
+    return field.field.widget.__class__.__name__ == "TextInput"
+
+@register.filter
+def is_textarea(field):
+    return field.field.widget.__class__.__name__ == "Textarea"
```

Comment:
are you sure you need these? can't you check for the fields name or something like that? like, "if this field is the body field, then make a custom input", not "if this field is a textarea, make a custom input"